R/build_update_taxon_list.R

```diff
@@ -129,79 +151,90 @@ build_update_taxon_list <- function(austraits, taxon_list, replace = FALSE) {
     dplyr::mutate(
       count_naturalised = stringr::str_count(.data$taxon_distribution, "naturalised"),
       count_n_and_n = stringr::str_count(.data$taxon_distribution, "native and naturalised"),
       count_states = stringr::str_count(.data$taxon_distribution, ",") + 1,
       establishment_means = ifelse(.data$count_naturalised > 0 & .data$count_n_and_n == 0, "naturalised", NA),
-      establishment_means = ifelse(.data$count_n_and_n > 0 | (.data$count_naturalised > 0 & .data$count_states > .data$count_naturalised), "native and naturalised", .data$establishment_means),
-      establishment_means = ifelse(.data$count_naturalised == 0 & .data$count_n_and_n == 0, "native", .data$establishment_means),
+      establishment_means = ifelse(
+        .data$count_n_and_n > 0 | (.data$count_naturalised > 0 & .data$count_states > .data$count_naturalised),
+        "native and naturalised", .data$establishment_means),
+      establishment_means = ifelse(
+        .data$count_naturalised == 0 & .data$count_n_and_n == 0,
+        "native", .data$establishment_means),
       establishment_means = ifelse(.data$taxon_rank %in% higher_ranks, NA, .data$establishment_means),
       taxon_distribution = ifelse(.data$taxon_rank %in% higher_ranks, NA, .data$taxon_distribution)
       ) %>%
-    dplyr::select(dplyr::all_of(c("aligned_name", "taxon_name", "taxon_rank", "taxonomic_status", "taxonomic_dataset", "taxon_name_alternatives",
-                                  "genus", "family", "binomial", "trinomial", "taxon_distribution", "establishment_means", "scientific_name",
-                                  "taxon_id", "taxon_id_genus", "taxon_id_family", "scientific_name_id", "aligned_name_taxon_id", "aligned_name_taxonomic_status")))
+    dplyr::select(dplyr::all_of(
+      c("aligned_name", "taxon_name", "taxon_rank", "taxonomic_status", "taxonomic_dataset",
+      "taxon_name_alternatives", "genus", "family", "binomial", "trinomial", "taxon_distribution",
+      "establishment_means", "scientific_name", "taxon_id", "taxon_id_genus", "taxon_id_family",
+      "scientific_name_id", "aligned_name_taxon_id", "aligned_name_taxonomic_status")))
+
+    # New taxon list
 
-    # New taxon list  
-  
    if (replace == TRUE) {
-      taxon_list_replace <- taxon_list_new %>% 
+
+      taxon_list_replace <- taxon_list_new %>%
         arrange(taxon_name, aligned_name) %>%
         distinct(taxon_name, aligned_name, .keep_all = TRUE)
-     
+
    } else {
-     
+
       taxon_list_replace <- taxon_list %>%
         # First bind rows for cleaned names not yet in AusTraits taxon_list.csv file
-        bind_rows(taxon_list_new %>% filter(!aligned_name %in% taxon_list$aligned_name)) %>%
+        bind_rows(taxon_list_new %>% dplyr::filter(!aligned_name %in% taxon_list$aligned_name)) %>%
         # Arrange by names - hopefully this will be best solution for keeping GitHub commits more transparent
         arrange(taxon_name, aligned_name) %>%
```

Comment:
can you also add `dplyr::` to the other places it is missing while you're at it? several `arrange` and `distinct`

Reply:
@ehwenk yes happy to do that! I'm not sure if there are similar scenarios of confusion for the other functions like with `filter` but happy to fix just in case.